### PR TITLE
feat(docs): Fix grunt build for UI Docs

### PR DESCRIPTION
**Pull request by nidzix**

### CODE REVIEW

**Comment by ioanpocol** [comment]
:+1:

**Comment by amagdas** [comment]
:+1:

File: client/app/docs.html [hunk 1]
```diff
@@ -8,25 +8,34 @@
...
+    <!-- build:js scripts/superdesk-docs-core.js -->
     <script src="scripts/superdesk/services/asset.js"></script>
-
-    <script src="scripts/config.js"></script>
-
     <script src="docs/module.js"></script>
+    <!-- endbuild -->
+
+    <!-- build:js scripts/superdesk-docs-main.js -->
+    <!-- endbuild -->
```
Comment (petrjasek) [inline client/app/docs.html]: i guess this should go after `<script>`s

Comment (nidzix) [inline client/app/docs.html]: no, because this is the way of making empty `superdesk-docs-main.js` which I need for `requirejs` task

Comment (petrjasek) [inline client/app/docs.html]: I see

**Pull request merged**